src/handlers/next/create-manifest-handler.ts

```diff
@@ -1,12 +1,13 @@
 import { NextApiHandler, NextApiRequest } from "next";
 
-import { getBaseUrl } from "../../headers";
+import { getBaseUrl, getSaleorHeaders } from "../../headers";
 import { AppManifest } from "../../types";
 
 export type CreateManifestHandlerOptions = {
   manifestFactory(context: {
     appBaseUrl: string;
     request: NextApiRequest;
+    schemaVersion: number;
```

Comment:
I dont think it should fallback to zero. I think it should be `number | null`. If field is not supported, then its not supported. No need for ambiguous values